pdctl/command/config_command.go

```diff
@@ -68,50 +121,80 @@ func showConfigCommandFunc(cmd *cobra.Command, args []string) {
 	fmt.Println(r)
 }
 
+func showAllConfigCommandFunc(cmd *cobra.Command, args []string) {
+	r, err := doRequest(cmd, configPrefix, http.MethodGet)
+	if err != nil {
+		fmt.Printf("Failed to get config: %s", err)
+		return
+	}
+	fmt.Println(r)
+}
+
 func setConfigCommandFunc(cmd *cobra.Command, args []string) {
 	if len(args) != 2 {
 		fmt.Println(cmd.UsageString())
 		return
 	}
+	var (
+		value interface{}
+		path  string
+	)
+	opt, val := args[0], args[1]
 
-	url := getAddressFromCmd(cmd, schedulePrefix)
-	var value interface{}
-	data := make(map[string]interface{})
+	f, ok := optionRel[opt]
```

Comment:
like `config set schedule <option> <value>` and `config set replicas <option> <value>`?  but we also need to distinguish the field type like string and []string. I think this way is ok, it is no need to adjust when I add a field in config.